### ISSUE TITLE
Analyzer errors in Dart 3.9 projects

### ISSUE THREAD

**Issue by masa-tokyo** [altive/altive_lints#107]
Hi,
Thank you very much for the great package!

I’d be glad if this package could become compatible with Dart 3.9 projects.
Please take a look at the description below :)

**Describe the bug**
Currently, when running `dart analyze` on CI for a Dart 3.9 project, the same issue occurs as described here:
https://github.com/dart-lang/sdk/issues/61168

```
Analyzing ....
ERROR: ERROR: An unexpected error was encountered by the Analysis Server.
Please file an issue at https://github.com/dart-lang/sdk/issues/new/choose with the following details:
Failed to start plugins
An error occurred while executing an analyzer plugin: Bad state: Failed to start the plugins.
#0      SocketCustomLintServerToClientChannel._checkInitializationFail.<anonymous closure> (package:custom_lint/src/v2/server_to_client_channel.dart:223)
<asynchronous suspension>
#1      Future.any.onValue (dart:async/future.dart:616)
<asynchronous suspension>
An unexpected error was encountered by the Analysis Server.
Please file an issue at https://github.com/dart-lang/sdk/issues/new/choose with the following details:
Loading dynamic library failed: not an ELF shared object
lib/custom_lint_client.dart is not an AOT snapshot, it cannot be run with 'dartaotruntime'
```

As the issue above suggests, the latest custom_lint (0.8.0) would fix the problem, while this altive_lints package seemingly still uses custom_lint 0.7.6.

**To Reproduce**
1. Set Dart version to 3.9 in pubspec.yaml (`sdk: ^3.9.0`)
2. Run `dart analyze` on GitHub Actions

**Expected behavior**
No errors like the ones mentioned above.

**Additional context**
The workaround is to use the `--no-use-aot-snapshot` flag, as [the changelog](https://github.com/dart-lang/sdk/blob/main/CHANGELOG.md#analyzer) suggests.
However, with this approach, the runtime becomes a bit longer, and warnings are not shown in the local IDE.

**Comment by riscait** [comment]
Thank you for opening the issue! It’s very helpful.
I’ll bump the version of custom_lint_builder and release a new version🚀

**Comment by riscait** [comment]
It’s been released! Please let us know if there’s anything else.
https://pub.dev/packages/altive_lints/changelog#1221

**Comment by masa-tokyo** [comment]
@riscait 
The latest version fixed the issue!!
Thank you very much for the quick fix 🙏
